frontend/src/extensions/extra-integrations/fork-awesome/fork-awesome-app-extension.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+import type { Linter } from '../../../components/editor-page/editor-pane/linter/linter'
+import { SingleLineRegexLinter } from '../../../components/editor-page/editor-pane/linter/single-line-regex-linter'
+import { AppExtension } from '../../base/app-extension'
+import { t } from 'i18next'
+
+export const forkAwesomeRegex = /<i class=["']fa fa-[\w-]+["'](?: aria-hidden=["']true["'])?\/?>(?:<\/i>)?/
```

Comment:
```suggestion
export const forkAwesomeRegex = /<i class=["'][\w\s]*fa-[\w-]+[\w\s-]*["'](?: aria-hidden=["']true["'])?\s*\/?>(?:<\/i>)?/
```

Reply:
This is also valid for example: `<i class="fa fa-flag fa-stack-1x fa-inverse"></i>`